.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
I don't believe we need this, looking at the main repository.

Reply:
It makes it pretty hard to work with this repository on Macs. They create this file everywhere and without this I have to manually get rid of it before every commit.

Reply:
Ah ok! thanks for the heads-up